pkg/configs/advisory/v2/document.go

```diff
@@ -131,13 +131,37 @@ var (
 
 // Get returns the advisory with the given ID. If such an advisory does not
 // exist, the second return value will be false; otherwise it will be true.
-func (advs Advisories) Get(id string) (Advisory, bool) {
+func (advs Advisories) Get(id string, aliases []string) (Advisory, bool) {
 	for _, adv := range advs {
 		if adv.ID == id {
 			return adv, true
 		}
 	}
 
+	for _, reqAlias := range aliases {
+		for _, adv := range advs {
+			for _, alias := range adv.Aliases {
+				if alias == reqAlias {
+					return adv, true
+				}
+			}
+		}
+	}
+
+	return Advisory{}, false
+}
+
+// Get returns the advisory with the given ID. If such an advisory does not
+// exist, the second return value will be false; otherwise it will be true.
+func (advs Advisories) GetAlias(id string) (Advisory, bool) {
```

Comment:
Like above, I don't think we should need to add this...

Reply:
yep, i think i forgot to drop this one